### ISSUE TITLE
add example of `verdi plugin list` command to the tutorial

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-core#4544]
### Is your feature request related to a problem? Please describe

`verdi plugin list` is an indispensable tool to figure out information about CalcJobs and Workflows from the command line - what inputs they need and, in particular, how they can fail and what the exit codes mean.

It is currently not used in the tutorial.

### Describe the solution you'd like

Use e.g. `verdi plugin list aiida.calculations` to show where the `arithmetic.multiply_add` entry point is taken from (which currently comes out of nowhere) and use `verdi plugin list aiida.calculations arithmetic.multiply_add` to show which inputs it needs.

@mbercx what do you think?

**Comment by mbercx** [comment]
I agree it is a very important tool, and might deserve a place in the basic tutorial. However, I'm a little worried about adding too much to this initial tutorial. If we have to show where the entry point comes from and also explain how to use `verdi plugin list` to see the inputs and exit codes - with examples - that will add quite a bit of content. We also don't explain exit codes in the tutorial, so that might be confusing at this point.

Currently the `verdi plugin list` command is explained first thing in the ["Launching a predefined workflow"](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/workflows.html#launching-a-predefined-workflow) section. It's also mentioned in the ["How to submit a calculation"](https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/run_codes.html#how-to-submit-a-calculation) section. So if the user continues after the tutorial to run a code or workflow, they should be introduced to the command quite quickly. Perhaps that is sufficient?

**Comment by ltalirz** [comment]
I just noticed new users not being aware of the command at all.

One could also do a compromise and use it just to figure out the name of the entry point (only running `verdi plugin list aiida.workflows`)

**Comment by mbercx** [comment]
> One could also do a compromise and use it just to figure out the name of the entry point (only running `verdi plugin list aiida.workflows`)

Sounds like a good compromise! I may also be overly worried about adding too much content to the basic tutorial. I'll work on it a bit and we'll see what works best.

**Comment by ltalirz** [comment]
There is still no mention of it in the basic tutorial

Section https://aiida.readthedocs.io/projects/aiida-core/en/latest/intro/tutorial.html#workflows mentions entry points for the first time, but they come out of the blue.

I suggest
- [ ] adding a mention of `verdi plugin list` 
- [ ] adding a link to https://aiida.readthedocs.io/projects/aiida-core/en/latest/howto/plugin_codes.html#registering-entry-points for users who are reading this tutorial wanting to create their own workflow  (e.g. @clarkszw was in this situation and was missing the link to where this entry point comes from)

@clarkszw if you feel comfortable making these two edits, a PR would be very welcome (just click "Edit this page" on the docs page).

**Comment by Clarkszw** [comment]
@ltalirz I think it will be easier if I just add some links that will be helpful for a beginner from my perspective. I have created a pull request:)
